pandas/io/formats/xml.py

```diff
@@ -15,9 +15,11 @@
     WriteBuffer,
 )
 from pandas.errors import AbstractMethodError
+from pandas.util._decorators import doc
 
 from pandas.core.dtypes.common import is_list_like
 
+from pandas.core import generic
```

Comment:
same comment this causes a circular import, instead use shared_docs which isn't a problem to import